homie-influx/src/influx.rs

```diff
@@ -79,6 +79,12 @@ fn point_for_property_value(
     if let Some(node_type) = node.node_type.to_owned() {
         point = point.add_tag("node_type", Value::String(node_type))
     }
+    if let Some(Datatype::Boolean) = property.datatype {
```

Comment:
Please add a comment here explaining why this is necessary, with a link to the Grafana issue.

Reply:
Done.